SilKit/source/services/logging/Logger.hpp

```diff
@@ -56,6 +139,10 @@ class Logger : public ILogger
     // Public interface methods
     //
     // ILogger
+    void Log(const LoggerMessage& msg) override;
+
+    void Log(const LogMsg& msg) override;
```

Comment:
Please create a new 'comment block' `// ILoggerInternal` (since these methods aren't present in the `ILogger` interface) and put them there.